index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./styles/style.css">
+        <!-- For social media icons: -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <div class="grid-container">
+            <header>
+                <h1 class="my-name"><a href="./index.html">WHIT SUNDBY</a></h1>
+                <nav class="navbar">
+                    <a href="./about.html">About</a></li>
+                    <a href="./portfolio.html">Portfolio</a></li>
+                    <a href="./blog.html">Essays</a></li>
+                </nav>
+            </header>
+            <main>
+                <br>
+                <section class="featured">
+                    <article class="card">
+                        <img src="./images/video_store.jpg" alt="A storefront sign that says 'VIDEO'." />
+                        <!-- <section class="card_text">
```

Comment:
Just a note, that these images really squish down when the screen size is small.